### PR TITLE
Power Monitoring and Control

### PR DESCRIPTION
**Pull request by m-mcgowan**
# Problem Context

Please see decomp and specs for NF8,NF11-13.

### CODE REVIEW

File: power-monitoring-and-control/firmware/src/SWAN.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
```
Comment (bsatrom) [inline power-monitoring-and-control/firmware/src/SWAN.txt:1]: I don't think we need this file in this repo

**Review by bsatrom — APPROVED** [review]
looks good, just some minor things!

File: power-monitoring-and-control/firmware/src/app.cpp [hunk 2]
```diff
@@ -0,0 +1,760 @@
...
+
+
+// Notefile/Note definitions
+#define	DATA_FILENAME			"power.qo"
+#define DATA_FIELD_APP          "app"
+#define DATA_FIELD_EVENT		"event"
+#define DATA_FIELD_INSTANCE		"instance"
+#define DATA_FIELD_VOLTAGE		"voltage"
+#define DATA_FIELD_CURRENT		"current"
+#define DATA_FIELD_POWER		"power"
```
[the remark this answers is not in the document]
Comment (m-mcgowan) [reply power-monitoring-and-control/firmware/src/app.cpp]: tabs vs spaces. fight! :-)

**Comment by m-mcgowan** [comment]
Thanks for the review @bsatrom. I made the changes you suggested and merged since they were mostly trivial.

**Pull request merged**